data-serving/scripts/export-data/functions/02-export/app.py

```diff
@@ -47,8 +49,8 @@ def export_chunk(skip, limit, num_cases, num_chunk, num_chunks, field_names, exc
             f"--limit={limit}"
     ]
     if exclude_sources:
-        query = json.dumps({"caseReference.sourceId": {"$nin": exclude_sources}})
-        mongoexport.append(f"--query='{query}'")
+        query['caseReference.sourceId'] = {"$nin": exclude_sources}
```

Comment:
Is `exclude_sources` here used for GISAID? If so this can now be removed.

Reply:
Good catch, removed